compiler/rustc_codegen_cranelift/example/mini_core_hello_world.rs

```diff
@@ -389,9 +389,7 @@ global_asm! {
 #[cfg(all(not(jit), target_arch = "x86_64"))]
 #[naked]
 extern "C" fn naked_test() {
-    unsafe {
-        naked_asm!("ret");
-    }
+    naked_asm!("ret")
 }
```

Comment:
Did we want to mark this one as `unsafe(naked)`?

Reply:
Good catch, looks like this is the CI failure